statsmodels/genmod/families/family.py

```diff
@@ -1451,10 +1455,28 @@ def loglike_obs(self, endog, mu, var_weights=1., scale=1.):
 
         Notes
         -----
-        This is not implemented because of the complexity of calculating an
-        infinite series of sums.
+        If eql is True, the Extended Quasi-Likelihood is used.  At present,
+        this method returns NaN if eql is False.  When the actual likelihood
+        is implemented, it will be accessible by setting eql to False.
+
+        References
+        ----------
+        http://support.sas.com/documentation/cdl/en/stathpug/67524/HTML/default/viewer.htm#stathpug_hpgenselect_details16.htm
         """
-        return np.nan
+        if self.eql == False:
+            # We have not yet implemented the actual likelihood
+            return np.nan
+
+        p = self.var_power
+        llf = np.log(2 * np.pi * scale) + p * np.log(endog) - np.log(var_weights)
+        llf *= -1 / 2
+
+        u = endog ** (2 - p) - (2 - p) * endog * mu ** (1 - p) + (1 - p) * mu ** (2 - p)
+        u *= var_weights / (scale * (1 - p)**2)
```

Comment:
When testing with sas's formula I realize they actually have a typo it is (1-p)*(2-p) not (1-p)**2

Reply:
Thanks, I will fix it.  It you have a separate PR with better tests we can go with yours.

Reply:
So I actually liked a paper by Kaas when I was studying this... 

https://lstat.kuleuven.be/research/seminars_events/files/3afmd/Kaas.PDF

Page 8 has his formula for EQL (formula 4). The deviance on page 7 (formula 3) should pretty much be our `_resid_dev` method. You could call the existing method instead of re-writing it. You just have to add on the `var_weights` piece and the 2 * pi * scale adjustment.

You might also want to note in the doctoring that I think this EQL function is only valid when `1 < p < 2`.

Reply:
Yes exactly, only valid between 1 < p < 2 because at 1 you would use poisson and at 2 gamma.